[163625759] ft getall parties

st9ph8n opened the pull request:
#### What does this PR do?
Has get endpoint resources
#### Description of Task to be completed?
Get political parties
#### How should this be manually tested?
install the app and dependencies, run npm start:dev, open post man and get res through endpoint /api/v1/parties
#### What are the relevant pivotal tracker stories?
163625759

st9ph8n committed:
Add eslint

st9ph8n committed:
Add nodemon

st9ph8n committed:
Add start:dev script

st9ph8n committed:
Merge pull request #1 from st9ph8n/ch-Setup-nodemon-163597245

[163597245]-Ch-setup-nodemon

st9ph8n committed:
Merge branch 'develop' into ch-setup-linter-163597041

st9ph8n committed:
added lint script

st9ph8n committed:
add proper lint script

st9ph8n committed:
updated to eslint style guide

st9ph8n committed:
Merge pull request #2 from st9ph8n/ch-setup-linter-163597041

[163597041]-ch-setup-linter

st9ph8n committed:
Merge pull request #3 from st9ph8n/ch-setup-linter-163597041

Ch setup linter 163597041

st9ph8n committed:
add testing dependencies

st9ph8n committed:
Add test script

st9ph8n committed:
Add some test setups

st9ph8n committed:
Add eslint config for test folder

st9ph8n committed:
Refactor test to pass test

st9ph8n committed:
Add test

st9ph8n committed:
Merge pull request #4 from st9ph8n/ft-Setup-testing-163601495

Ft setup testing 163601495

st9ph8n committed:
Add datastores

st9ph8n committed:
Add models

st9ph8n committed:
Merge pull request #6 from st9ph8n/ft-Add-model-datastore-163604541

 #163604541 Ft add models datastore

st9ph8n committed:
Add controllers

st9ph8n committed:
Merge pull request #7 from st9ph8n/ft-Add-controller-163608081

163608081 Add controllers

st9ph8n committed:
Add controllers

st9ph8n committed:
Add dataStore

st9ph8n committed:
Add routes

st9ph8n committed:
implement routes on app.js

st9ph8n committed:
Add body-parser

st9ph8n committed:
Updated port number

st9ph8n committed:
Merge pull request #8 from st9ph8n/ft-Add-endpoints-163609942

163609942 Ft add endpoints

st9ph8n committed:
Revert "163609942 Ft add endpoints"

st9ph8n committed:
Merge pull request #10 from st9ph8n/revert-8-ft-Add-endpoints-163609942

Revert "163609942 Ft add endpoints"

st9ph8n committed:
feat(createparties): create political parties
- endpoint to create political party
-add created party to datastore
[Delivers 163620904]

st9ph8n committed:
chore(app & package): Updated app.js and package.json
-added middleware to link proper endpoint name to create party Router
[Delivers163620904]

st9ph8n committed:
chore(party.js): Add save method
-ensure party is saved after created
[Delivers 163620904]

st9ph8n committed:
Delete controller

st9ph8n committed:
Merge pull request #13 from st9ph8n/ch-remove-controller-163624217

[163624217]Delete controller folder

st9ph8n committed:
feat(partycontroller & parties) implement create party
- implement create party endpoint an setup route
-add new party to datastore
[Delivers 163620904]

st9ph8n committed:
Merge pull request #12 from st9ph8n/ft-create-politicalParty-163620904

163620904 Ft create political party

st9ph8n committed:
feat(partyController,party,parties): implement Get endpoint
- Get parties from datastore
[Delivers 163625759]

kenware commented on controllers/partiesController.js:
```diff
@@ -0,0 +1,28 @@
+    if (party.save()) {
+      res.json({
+        status: 201,
+        data: party,
```
This should be an array of party object

kenware commented on models/vote.js:
```diff
@@ -0,0 +1,19 @@
+const voteStore = require('../dataStore/votes');
+
```
What is vote model doing in a get party endpoints

kenware commented on models/user.js:
```diff
@@ -0,0 +1,19 @@
+
+class User {
+  static all() {
+    return userStore;
```
What is user model doing in a get party endpoint

st9ph8n committed:
chore(modelfolder and partycontroller) Exclusively party endpoint
-Deleted files from model folder and made update to partycontroller
-[Delivers 163625759]
